openpyxl-stubs/worksheet/worksheet.pyi

```diff
@@ -128,7 +129,7 @@ class Worksheet(_WorkbookChild):
     def append(self, iterable: Any) -> None: ...
     def calculate_dimension(self) -> str: ...
     def cell(
-        self, row: int, column: int, value: Optional[int] = ...
+        self, row: int, column: int, value: Optional[Union[*KNOWN_TYPES]] = ...
```

Comment:
Have you tested that? When I try it like this, it fails:

```
>>> from typing import Union
>>> from openpyxl.cell.cell import KNOWN_TYPES
>>> Union[*KNOWN_TYPES]
  File "<stdin>", line 1
    Union[*KNOWN_TYPES]
                      ^
SyntaxError: invalid syntax
```

I think you might be able to to that without the `*`

Reply:
Oh, good question. I will investigate that when I have some time.

Reply:
Would it be OK for you if I adjusted this pr a bit (if necessary) before merging?

I want to give you credit, but I also want to make sure everything is correct.

Reply:
Thanks! Absolutely—I'm still new to type hinting.
I like using it because it helps me find bugs in my code early rather than late, but sometimes I get lost in the abstractions...

Reply:
```suggestion
        self, row: int, column: int, value: Optional[Union[int, str, bytes, datetime.datetime, datetime.date, datetime.time, datetime.timedelta, int, float, Decimal, ]] = ...
```

Reply:
Sadly, there is no better way than adding all types explicitly.

Reply:
`int` appears twice (doesn't really matter in a `Union`, but...), and both `datetime` and `Decimal` are not imported.
Do the lack of imports matter?

Reply:
All of those issues were not intentional. Yes, not importing datetime and Decimal does matter. Thank you for being careful! :+1: